server/app.js

```diff
@@ -25,6 +28,8 @@ app.use(cors({ credentials: true, origin: true }));
 app.use(express.json());
 
 // routes
+app.use("/api/v1/levels", levelRouter);
+app.use("/api/v1/frames", frameRouter);
```

Comment:
Are the endpoints supposed to be in singular or plural? If plural seems to be more of a conventional way (ref: https://gearheart.io/blog/restful-api-design-best-practices/), then this stands and I need to update my code for the other part).

Reply:
Changed mine so can consider this resolved.